reactapp/src/components/UploadImages.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component, createRef} from 'react';
+import Dropzone from 'react-dropzone';
+import "../styles/image_uploader.css";
+import {post} from 'axios';
+
+//Setup for dropzone component. createRef is for creating access/reference to the HTML page's DOM
```

Comment:
I'm not familiar with Javascript and its possible toolchains but, is there a way to create a documentation comment akin to Python's docstrings which we could build and host with Sphinx?

Reply:
Maybe look into this https://www.docz.site/

Reply:
Or maybe this is more relevant https://milmazz.uno/article/2014/08/27/how-to-document-your-javascript-code/

Reply:
Going with this - https://github.com/styleguidist/react-styleguidist . Seems to be a well supported React Library